### ISSUE TITLE
meaning of the State transition function

### ISSUE THREAD

**Issue by wzhang1** [peter-moran/highway-lane-tracker#3]
In line 339 of windows.py, what's the meaning of 0.5 for velocity? I change it to 0.0 and the result seems not changed?

**Comment by peter-moran** [comment]
That is a bit of an under documented tweak I made.

`self.kf.F` is the Kalman update function. It describes what we think the next state (the horizontal position of the window) of the system will be based on the last state. In this particular configuration, it is saying:

* The first line of the matrix is stating that I expect the next position to be at the previous position + previous velocity * time.

* The second line is stating that I expect the velocity to be about half the previous state. This is essentially like friction and theoretically helps the tracker from overshooting.

**Comment by wzhang1** [comment]
I agree with the explanation of the meaning for the second line, if you change it to 0.0, your model will be extremely simplified and I did not see the obvious different in the 3 udacity videos.(I tried ion other videos but hit the this line(TODO: Do something if not enough windows to fit. Most likely fall back on old measurements.)), so not sure if the velocity term have effects
